db/db_local/backup.go

```diff
@@ -79,7 +112,7 @@ func takeBackup(ctx context.Context, config *dbStartConfig) error {
 }
 
 // startDatabaseInLocation starts up the postgres binary in a specific installation directory
-func startDatabaseInLocation(ctx context.Context, location string) (*dbStartConfig, error) {
+func startDatabaseInLocation(ctx context.Context, location string) (*oldDbRunningConfig, error) {
```

Comment:
I meant to query thgis in the other PR - why is dbStartConfig renamed to oldDbRunningConfig?

Reply:
done in the other one